packages/valory/agents/oracle/aea-config.yaml

```diff
@@ -94,9 +94,10 @@ models:
     args:
       url: ${SKILL_ORACLE_ABCI_MODELS_PRICE_API_ARGS_URL:str:https://api.coinbase.com/v2/prices/BTC-USD/buy}
       api_id: ${SKILL_ORACLE_ABCI_MODELS_PRICE_API_ARGS_API_ID:str:coinbase}
-      parameters: ${SKILL_ORACLE_ABCI_MODELS_PRICE_API_ARGS_PARAMETERS:list:none}
+      parameters: ${SKILL_ORACLE_ABCI_MODELS_PRICE_API_ARGS_PARAMETERS:list:null}
```

Comment:
Is this necessary?

Reply:
For `None` type to be read from `python`, `null` has to be used in `yaml`. I would suggest that we also check for `none`s in all our configurations.